ci/build-loop.sh

```diff
@@ -173,6 +175,13 @@ if pushd "$PR_REPO_CHECKOUT"; then
   popd
 fi
 
+# Nomad runs this script inside a cgroup managed by it, so it can clean up
+# properly when we exit (or are killed), and it can track CPU/RAM usage.
+# Run our Docker builds inside the same cgroup so they're included too.
+if cgroup=$(sed -rn '/:freezer:/{s/.*:freezer:(.*)/\1/p;q}' "/proc/$$/cgroup"); then
```

Comment:
Isn't this linux specific?

Reply:
Yes it is, that's why it's inside the `if`.

On Mac, the `sed` command will fail and we won't change `DOCKER_EXTRA_ARGS` (though of course, we don't run in Docker on Mac anyway).